### DISCUSSION TITLE
magick: no decode delegate for this image format `WMF' @ error/constitute.c/ReadImage/556.

### DISCUSSION THREAD

**Discussion by luojc0517** [ImageMagick/ImageMagick#1903] [General] [answered]
### Environment
macOS Mojave 10.14.5 (18F132)

### Error Message
magick: no decode delegate for this image format `WMF' @ error/constitute.c/ReadImage/556.

### Screenshot

![image](https://user-images.githubusercontent.com/13957771/79580320-6a7dd700-80fb-11ea-8f76-108b1fd55bc1.png)

**Comment by luojc0517** [comment, marked as answer]
### Solution
`brew reinstall imagemagick --with-libwmf`

**Comment by synoptase** [comment]
For anyone landing here, proposed solution no longer works since: https://github.com/Homebrew/brew/pull/5274

One needs to edit the formula: https://apple.stackexchange.com/a/352928

```sh
brew edit imagemagick
```

- Add `--with-wmf` and remove `--without-wmf`

```sh
brew reinstall --build-from-source imagemagick
```

**Reply by elliotberry** [reply]
Thank you for this my dude

**Reply by jocapello** [reply]
Does this solution still work? The reinstall seems to ignore the with-wmf as building from source isnt supported
